### ISSUE TITLE
Automate registration of routes for analysis services APIs into Unity API Gateway

### ISSUE THREAD

**Issue by jjacob7734** [unity-sds/unity-analytics-dapa#15]
The routes for the DAPA API for Unity analysis services need to be registered in the shared API Gateway.  Installation of these routes should be automated with Terraform.

Acceptance criteria:

- [x] Time series route in API-Gateway in `Unity-Dev` venue
- [x] Test API by calling SDAP time series

**Comment by GodwinShen** [comment]
@jjacob7734 any updates on this ticket?  Also, you mentioned that "Installation of these routes should be automated" but that isn't explicitly called out in your acceptance criteria.  How do you plan to verify that automated route installation is happening and working correctly?

**Comment by GodwinShen** [comment]
@jjacob7734 Can you please provide an update on this ticket?  It was originally planned to complete in the last 24.1.2 sprint but is still listed as "In Progress"?

**Comment by jjacob7734** [comment]
This ticket was created as a subtask of https://github.com/orgs/unity-sds/projects/3/views/9?filterQuery=U-AS&pane=issue&itemId=51945477.  The objective is to work out an issue we encountered in automating the deployment.  This has been solved and the root cause of earlier problems was that we were calling version 2 of the AWS service for doing the deployment and we needed to use version 1.  The automation of the API installation into API Gateway is now an acceptance criterion in that ticket.  I am closing this ticket.
